website/versioned_docs/version-5.x/ecosystem-angular.md

```diff
@@ -781,3 +781,33 @@ const lifecycles = singleSpaAngular({
 ```
 
 > ⚠️ `single-spa-angular@4.x` requires calling `getSingleSpaExtraProviders` function in applications that have routing. Do not call this function in _zone-less_ application.
+
+### Parcels
+
+In order to consume parcels created in other frameworks (vue, react) in angular, there is a angular parcel component available which can be used to fetch the vue or react parcel and add it to the DOM in angular. The link to the parcel component https://github.com/ricardofct/single-spa-angular/blob/angular-parcel/src/browser-lib/parcel-lib/parcel.component.ts. Copy this component and paste it in your angular code.
```

Comment:
This isn't correct. Parcels have been added to single-spa-angular's public API, see here: https://github.com/single-spa/single-spa-angular/blob/master/src/src/public_api.ts#L3-L4
You should instead be able to `import { ParcelModule, ParcelComponent } from 'single-spa-angular'` but I don't have an Angular app to validate that at the moment. Will you try out that syntax and update these docs?